app/controllers/admin/change_owner_controller.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+module Admin
+  # Controller for changing ownership of resources
+  class ChangeOwnerController < Admin::ApplicationController
+    def new
+      authorize!
+
+      @data = data
+      @change_owner_form = Admin::ChangeOwnerForm.new(content_id: params[:content_id])
```

Comment:
What is the content_id used for?

Reply:
Good catch - at some point I think it may have been needed (or I saw it used for) the work_form, but it appears unused. Cleaning up.